### ISSUE TITLE
RN app crash on version 0.47.0 with ios on debug mode

### ISSUE THREAD

**Issue by softCompound** [facebook/react-native#16161]
### Is this a bug report?

Potentially yes.

### Have you read the [Contributing Guidelines](https://facebook.github.io/react-native/docs/contributing.html)?

(Write your answer here.)

### Steps to Reproduce

(Write your steps here:)

1. run the app with react-native run-ios
2. cmd+d and select debug js
3. simulator shows a message on red screen

### Expected Behavior

(Write what you thought would happen.)

### Actual Behavior

(Write what happened. Add screenshots!)

### Reproducible Demo

(Paste the link to an example project and exact instructions to reproduce the issue.)

![rn-issue-ios](https://user-images.githubusercontent.com/20586602/31058485-43389720-a6ec-11e7-8756-09cabddeb1eb.png)

**Comment by softCompound** [comment]
Environment:
  OS:  macOS Sierra 10.12.6
  Node:  6.11.1
  Yarn:  Not Found
  npm:  3.10.10
  Watchman:  4.7.0
  Xcode:  Xcode 9.0 Build version 9A235
  Android Studio:  2.3 AI-162.4069837

Packages: (wanted => installed)
  react: 16.0.0-alpha.12 => 16.0.0-alpha.12
  react-native: ^0.47.0 => 0.47.0

**Comment by rohitgoyal** [comment]
+1

**Comment by kelset** [comment]
From my experience this kind of error is not a bug, can you share the code of the file where you have the ScrollView?

have you tried wiping the node modules and reinstalling them?

**Comment by softCompound** [comment]
Hi,
Thanks for the response. The problem has now been fixed. I deleted the node_modules and then cleaned the npm cache. Thereafter, reinstalled the modules via npm and now i am running on react-native v0.49.1.

It seems in many cases deleting the node_modules and reinstalling the modules helps solve the issues. But how? It would be nice if anyone could help me understand the co-relation between them.
